### DISCUSSION TITLE
Hello! I am new here!

### DISCUSSION THREAD

**Discussion by Noblesse013** [community/community#109930] [GitHub Education]
### Hi everyone,

I am Fiona! I just got selected to be GitHub Global Campus student! I am very much eager to know how can i use the students developer pack and excited to learn new things.

**Comment by drdixit** [comment]
First of all, you need two things
1 school or college id (you need to upload that for student account activation)
2 school or college email (846498544@testuniversity.ac.in it looks something like this provided by your school / collage)

Somethings keep that in mind
Your first name and last name supposed to same as shown in school/collage id
your GitHub account needs to have your university/school email

(Note: Don't just create new account with university email if you don't want to, you can just add your university/school email as secondary email in your current account, and it works that's how I did it)

1 make sure your account first name and last name as shown in school/university id (this is not username you can keep username whatever you want)
2 create new account with university/school email **or** add that email to your existing account as secondary email

after that Go to this [link](https://education.github.com/pack)

go there fill up all the details, wait for some days it takes little time to verify, and you have it.


[more information you can find it here](https://docs.github.com/en/education/about-github-education/github-education-for-students/apply-to-github-education-as-a-student)

after that you can just click on this [page](https://education.github.com/pack) and get affiliated links and easily activate all the student benefits.
some of them is quite tricky because different companies have different criteria. but 70 to 80% you can just click and activate.

**Comment by kenil-gopani** [comment]
hi i am also new
